src/Compilers/Core/CodeAnalysisTest/MetadataReferences/MetadataReferenceTests.cs

```diff
@@ -55,7 +55,7 @@ public void CreateFrom_Errors()
         [Fact]
         public void CreateFromImage()
         {
-            var r = MetadataReference.CreateFromImage(TestResources.NetFX.v4_0_30319.mscorlib);
+            var r = MetadataReference.CreateFromImage(ResourcesNet451.mscorlib);
```

Comment:
>ResourcesNet451 [](start = 54, length = 15)

Why not keep the original `TestResources` static class or namespace? E.g. `TestResources.Net451`

Reply:
I initially tried that but eventually moved away because I wanted a clean separation between what was coming from the proprietary resource DLL and what was coming from the "new" approach.

Reply:
How about `TestMetadata`? Or some other container type like that. Having it directly under Utilities namespace puts names like `BuildExtensions` and `ValueTask` out of context.

Reply:
That name works for me.